src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -1155,12 +1155,44 @@ var record = dcScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(
         assertThat(project.getName(), is(equalTo("Norges forskningsråd")));
     }
 
+    @Test
+    void shouldScrapeProjectSeparatedBySlash() {
+        var type = toDcType("Journal article");
+        var citationContainingJournalName = new DcValue(Element.RELATION, Qualifier.PROJECT, "EC/H2020/727610");
+        var dublinCore = DublinCoreFactory.createDublinCoreWithDcValues(List.of(type, citationContainingJournalName));
+        var record = dcScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(null), "uit");
+        var project = record.getProjects().iterator().next();
+        assertThat(project.getIdentifier(), is(equalTo("727610")));
+        assertThat(project.getName(), is(equalTo("EC/H2020")));
+    }
+
+    @Test
+    void shouldScrapeProjectSeparatedBySlashAndColon() {
```

Comment:
Might have created parameterized tests for expectations of Slash-y/Colon-y variants. Like `shouldCreateProjectWhenStringContainsSlash` etc.